canonical-web-and-design/cn.ubuntu.com
# Dynamic about

### Done

Made the /about page use the CMS
### QA

`make import-new-pages`
`make run`
Navigate to /about and see that the correct content is there
### Card

https://canonical.leankit.com/Boards/View/111185042/118530727
